Sources/Sentry/SentryUIViewControllerSwizziling.m

```diff
@@ -163,7 +184,7 @@ + (void)swizzleLoadView:(Class)class isNib:(BOOL)isNib
     // for a nib file and doesn't load a view. This would lead to crashes as no view is loaded. As a
     // workaround, we skip swizzling the loadView and accept that the SKD doesn't create a span for
     // loadView if the UIViewController doesn't implement it.
-    if (isNib) {
+    if (![class respondsToSelector:@selector(loadView)]) {
         return;
     }
```

Comment:
h: this does not work. The object `class` does not respond to `loadView`

What you are looking for is something like this:
```suggestion
    SEL selector = NSSelectorFromString(@"loadView");
    
    IMP viewControllerImp= class_getMethodImplementation([UIViewController class], selector);
    IMP classLoadViewImp = class_getMethodImplementation(class, selector);
    if (viewControllerImp == classLoadViewImp) {
        return;
    }
```

However, I'm not sure about the performance of this solution.

Reply:
Good point, thanks. After running a time profiler the `swizzleLoadView` didn't even show up, so I think the performance is fine. Anyway, it should be a simple lookup.